dvc/main.py

```diff
@@ -16,6 +17,16 @@
 
 logger = logging.getLogger("dvc")
 
+footer = (
```

Comment:
I would create this message only when it's needed. Maybe some `show_troubles_footer()` func.

Reply:
@Suor There is no point to do that. This message is meant to be static.

Reply:
@efiop but it's created on each run for no purpose.

Reply:
Not sure I follow, could you elaborate?

Reply:
Ah, I guess you mean ColorFormatter instance:slightly_frowning_face: Good call.

Reply:
Ah, wait, it is not. You meant that it is created for each `dvc` CLI command run. That is totally fine, don't see anything wrong with it.

Reply:
I mean we import `colorama` and construct the message even when we'll never need it. I was thinking about startup time. However, since we import `colorama` in `dvc.logger` anyway then it probably doesn't matter for now.